lib/src/model/game/game_bookmark_provider.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:lichess_mobile/src/model/common/id.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'game_bookmark_provider.g.dart';
+
+/// A provider to store the bookmark value of a game when it was changed by the user.
+@Riverpod(keepAlive: true)
+class GameBookmark extends _$GameBookmark {
```

Comment:
This provider looks dubious, I think it is not necessary and this state can be store elsewhere (probably a transient widget state is enough).